flowmachine/tests/test_query_join.py

```diff
@@ -15,11 +14,10 @@
 from flowmachine.features import daily_location
 from flowmachine.core.query import Query
 from flowmachine.core.custom_query import CustomQuery
-from pandas import DataFrame
 
 
 # Define a class that gives us some sample data to join on
-class limit(Query):
+class Limit(Query):
```

Comment:
This seems like a slightly non-descriptive name. Should we change it to e.g. `DailyLocStub` or similar?